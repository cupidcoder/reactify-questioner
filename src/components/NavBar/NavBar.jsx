import React from 'react';
import ListItem from '../ListItem/ListItem';

const NavBar = ({ items }) => (
  <ul>
    {items.map(item => <ListItem key={item.no} text={item.text} path={item.path} clicked={item.clicked} />)}
  </ul>
);

export default NavBar;
