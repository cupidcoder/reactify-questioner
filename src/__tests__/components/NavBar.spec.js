import React from 'react';
import { shallow } from 'enzyme';
import NavBar from '../../components/NavBar/NavBar';

describe('NavBar', () => {
  it('Should correctly save snapshot of the NavBar', () => {
    const items = [
      { no: 1, text: 'Sign in', path: '/signin' },
      { no: 2, text: 'Sign up', path: '/signup' }
    ];
    const wrapper = shallow(<NavBar items={items} />);
    expect(wrapper).toMatchSnapshot();
  });
});
