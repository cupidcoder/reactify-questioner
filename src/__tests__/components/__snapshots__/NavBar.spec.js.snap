// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Should correctly save snapshot of the NavBar 1`] = `ShallowWrapper {}`;
